File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py

def solution(num_list):
    answer = 0
    oddNum = 0
    evenNum = 0
    oddStr = ""
    evenStr = ""
    for i in num_list:
        if i % 2 == 0:
            evenStr += str(i)
        else:
            oddStr += str(i)
    oddNum = int(oddStr)
    evenNum = int(evenStr)
    answer = oddNum + evenNum
    return answer

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int oddNum = 0;
        int evenNum = 0;
        String oddStr = "";
        String evenStr = "";
        for(int num: num_list){
            if(num % 2 == 0){
                evenStr += String.valueOf(num);
            }
            else{
                oddStr += String.valueOf(num);
            }
        }
        oddNum = Integer.parseInt(oddStr);
        evenNum = Integer.parseInt(evenStr);
        answer = oddNum + evenNum;
        return answer;
    }
}